GoogleSignIn/Sources/GIDSignInPreferences.m

```diff
@@ -39,6 +50,36 @@
   return [NSString stringWithFormat:@"gid-%@", @STR(GID_SDK_VERSION)];
 }
 
+// Get the current Apple execution environment.
+NSString* GIDEnvironment(void) {
```

Comment:
Is this worth a unit test?

Reply:
Yes it is.  Not sure how to cover all the scenarios here, but we can check a few of them.